### PR TITLE
Adjust Additional Dependencies to remove windowsapp.lib so winrtact.dll can work on win8.1

### PR DESCRIPTION
**Pull request by kythant**
Adjust Additional Dependencies to remove windowsapp.lib so winrtact.dll can work on win8.1
Fixes this error
image

In addition, addressing:
Win8.1 currently errors with Error 0x8007109A – This operation is only valid in the context of an app container.
This is caused by the app container check that exists in windows 8.1 but not on later versions. 
To fix this, set linker option of your component to have /APPCONTAINER:NO

https://docs.microsoft.com/en-us/cpp/build/reference/appcontainer-windows-store-app?view=vs-2019
https://docs.microsoft.com/en-us/windows/uwp/debug-test-perf/windows-app-certification-kit-tests#appcontainercheck

### CODE REVIEW

File: src/UndockedRegFreeWinRT/Nuget/readme.txt [hunk 1]
```diff
@@ -47,6 +47,24 @@ Your application manifest should have the same name as, and be side by side to y
 Starting with Windows 10 19h1, the operating system has this feature by default and will use this information directly. Be sure to  target your application for windows so that the package automatically disables itself when this feature is available by default.
 https://docs.microsoft.com/en-us/windows/win32/sysinfo/targeting-your-application-at-windows-8-1
 
+
+Windows 8.1 Support (10/16/2020)
+Undocked egFreeWinRT can now activate types down to Windows 8.1. However, there are some minor things to account for on Windows 8.1.
```
Comment (Felix-Dev) [inline src/UndockedRegFreeWinRT/Nuget/readme.txt:52]: "Undocked egFreeWinRT" -> "Undocked RegFreeWinRT"

File: src/UndockedRegFreeWinRT/Nuget/readme.txt [hunk 2]
```diff
@@ -47,6 +47,24 @@ Your application manifest should have the same name as, and be side by side to y
 Starting with Windows 10 19h1, the operating system has this feature by default and will use this information directly. Be sure to  target your application for windows so that the package automatically disables itself when this feature is available by default.
 https://docs.microsoft.com/en-us/windows/win32/sysinfo/targeting-your-application-at-windows-8-1
 
+
+Windows 8.1 Support (10/16/2020)
+Undocked egFreeWinRT can now activate types down to Windows 8.1. However, there are some minor things to account for on Windows 8.1. 
+
+1) On Windows 8.1. you may encounter 0x8007109A (This operation is only valid in the context of an app container) error when activating your type. This is caused by the app container check that exists in windows 8.1 but not on later versions.
+This can be fixed by, setting the linker option of your component that you're activating to /APPCONTAINER:NO
```
Comment (Felix-Dev) [inline src/UndockedRegFreeWinRT/Nuget/readme.txt:55]: That comma looks out of place to me.

File: src/UndockedRegFreeWinRT/README.md [hunk 3]
```diff
@@ -48,6 +48,21 @@ Example application manifest:
 
 Starting with Windows 10 19h1, the operating system has this feature by default and will use this information directly. Be sure to use [Windows version targeting](https://docs.microsoft.com/en-us/windows/win32/sysinfo/targeting-your-application-at-windows-8-1) so that the package automatically disables itself when this feature is available by default.
 
+### Windows 8.1 Support (10/16/2020)
+Undocked RegFreeWinRT can now activate types down to Windows 8.1. However, there are some minor things to account for on Windows 8.1. 
+
+1) On Windows 8.1. you may encounter 0x8007109A (This operation is only valid in the context of an app container) error when activating your type. This is caused by the app container check that exists in windows 8.1 but not on later versions.
+This can be fixed by, setting the linker option of your component that you're activating to /APPCONTAINER:NO
```
Comment (Felix-Dev) [inline src/UndockedRegFreeWinRT/README.md:55]: Same as above.

File: src/UndockedRegFreeWinRT/Nuget/readme.txt [hunk 4]
```diff
@@ -47,6 +47,24 @@ Your application manifest should have the same name as, and be side by side to y
 Starting with Windows 10 19h1, the operating system has this feature by default and will use this information directly. Be sure to  target your application for windows so that the package automatically disables itself when this feature is available by default.
 https://docs.microsoft.com/en-us/windows/win32/sysinfo/targeting-your-application-at-windows-8-1
 
+
+Windows 8.1 Support (10/16/2020)
+Undocked egFreeWinRT can now activate types down to Windows 8.1. However, there are some minor things to account for on Windows 8.1. 
+
+1) On Windows 8.1. you may encounter 0x8007109A (This operation is only valid in the context of an app container) error when activating your type. This is caused by the app container check that exists in windows 8.1 but not on later versions.
```
Comment (Felix-Dev) [inline src/UndockedRegFreeWinRT/Nuget/readme.txt:54]: Does "[...] you may encounter error 0x8007109A (This operation...)" sound better?

Comment (kythant) [reply src/UndockedRegFreeWinRT/Nuget/readme.txt:54]: Yea kinda!

**Review by BenJKuhn — APPROVED** [review]

**Comment by jonwis** [comment]
https://docs.microsoft.com/en-us/windows/win32/apiindex/windows-apisets has some details about how to use APISets on Windows 8 & 8.1 ... would these help more generally?

**Comment by kythant** [comment]
Yup! I've referenced this for the PR.

**Pull request merged**